UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -74,9 +77,18 @@ public class POSFileDetails
             public string Sha1FileMimeType { get; set; }
             public string ZipFileMimeType { get; set; }
             public string PosDVDVolumeIdentifier { get; set; }
+            public string AioExchangeSetBatchId { get; set; }
+            public string InfoFolderName { get; set; }
+            public string AioFolderName { get; set; }
+
+        }
+
+        public class AioWebjobApiConfiguration
```

Comment:
Please keep 1 class per file

Reply:
As discussed and agreed, It is a existing file which is referred for more than 20+ tests, we just added few lines of code w.r.t AIO. If we plan to change the file than we need to change the previous tests as well as the new one. We will Take care of the above comment if we add anything new. Thanks